src/main/java/me/twostinkysocks/boxplugin/manager/MarketManager.java

```diff
@@ -259,7 +463,192 @@ public void cancelDeposit(InventoryClickEvent e, Player p) {
         openGui(p);
     }
 
-    public void openWithdrawMenu(Player p) {
+    public void plus16(InventoryClickEvent e, Player p) {
```

Comment:
for these functions, create a new function called addBal or something, and pass in the amount to add or subtract so you only have to write one